master/buildbot/changes/gerritchangesource.py

```diff
@@ -387,6 +433,14 @@ def eventReceived(self, event):
             yield self.master.db.state.setState(self._oid, 'last_event_ts', event['eventCreatedOn'])
         return res
 
+    @defer.inlineCallbacks
+    def getFiles(self, change, patchset):
+        res = yield self._http.get("/changes/%s/revisions/%s/files/" % (change, patchset))
+        res = yield res.content()
+
+        files = json.loads(bytes2unicode(res.splitlines()[1]))
```

Comment:
do you need this variable here?  why not just `return list(...)`?

Reply:
Personal style I guess. I often add breakpoints between lines in which case it helps to have a variable ready for inspection. There's a bit too much happening in a line like `list(json.loads(bytes2unicode(res.splitlines()[1])))` - I'll break it down further.